grails-bootstrap/src/main/groovy/org/grails/config/NavigableMap.groovy

```diff
@@ -488,4 +489,172 @@ class NavigableMap implements Map<String, Object>, Cloneable {
     boolean equals(Object obj) {
```

Comment:
@paulk-asert This class is tagged with both `@EqualsAndHashCode` and it then implements it's own equals / hashcode.  Do you know if the groovy transformation will have any effect?